testing/state-transition/state-transition.go

```diff
@@ -134,12 +134,16 @@ func SetupTestState(t *testing.T, cs chain.Spec) (
 		nodemetrics.NewNoOpTelemetrySink(),
 	)
 
-	ctx := &transition.Context{
-		Context:                 context.Background(),
-		SkipPayloadVerification: true,
-		SkipValidateResult:      true,
-		ProposerAddress:         dummyProposerAddr,
-	}
+	ctx := transition.NewTransitionCtx(
+		context.Background(),
+		0, // time
+		dummyProposerAddr,
+	).
+		WithVerifyPayload(false).
+		WithVerifyRandao(false).
+		WithVerifyResult(false).
```

Comment:
technically `WithVerifyRandao` should be true cause SkiValidateRandao was set to the default value of false.
However the intention of the tests was to skip these checks at all and focus on the state transition, so it's ok to not verify randao (and this PR makes it more explicit that it currently is)